Userland/Libraries/LibWeb/Painting/StackingContext.cpp

```diff
@@ -604,6 +596,21 @@ Optional<HitTestResult> StackingContext::hit_test(CSSPixelPoint position, HitTes
     return {};
 }
 
+bool StackingContext::is_hit_in_hidden_overflow(CSSPixelPoint transformed_position) const
```

Comment:
The negated logic of this function feels a little confusing to me. 
Can't this just be `StackingContext::is_hit` or `StackingContext::contains_point` and return `false` when it's not hit. Also as far this function is concerned it just a `position` not a `transformed_position`.

Reply:
I've changed it to `StackingContext::can_potentionally_hit`.

We don't know if we can hit without going through the whole hit test logic, however we can tell if we won't hit if we're inside the overflow for the axis which is hidden.

Reply:
:+1: I like that a lot more! Feel free to resolve these comments now.